grpc/src/main/java/com/linecorp/armeria/client/grpc/GrpcWebUtil.java

```diff
@@ -73,8 +77,15 @@
      * }</pre>
      */
     @Nullable
-    public static HttpHeaders parseTrailers(HttpData response) {
+    public static HttpHeaders parseTrailers(ClientRequestContext ctx, HttpData response) {
+        requireNonNull(ctx, "ctx");
         requireNonNull(response, "response");
+        final SerializationFormat serializationFormat =
+                ctx.log().ensureAvailable(RequestLogProperty.SCHEME).scheme().serializationFormat();
```

Comment:
How about adding `@throws` to the Javadoc?

Reply:
Added. 😉